tools/mcsema_disass/ida7/get_cfg.py

```diff
@@ -26,6 +26,10 @@
 import itertools
 import pprint
 
+### fix mcsema external symbol
+from elftools.elf.elffile import ELFFile
```

Comment:
Can you add `elftools` as a dependency to the README along with any instructions for fetching dependencies.